yt/go/yt/interface.go

```diff
@@ -921,6 +927,35 @@ type AdminClient interface {
 		options *RemoveMemberOptions,
 	) (err error)
 
+	// http:verb:"set_user_password"
+	// http:params:"user","new_password_sha256","current_password_sha256"
+	SetUserPassword(
```

Comment:
Why did you miss ListUserTokens method? It makes sense to support all auth-related methods together.

Reply:
Well, I've only added the ones I needed, but I can add one more, I think.

Reply:
done